README.md

```diff
@@ -1,2 +1,63 @@
-# rfcs
-Repo for managing Requests For Comments (RFCs) for the ESLint project
+# ESLint RFCs
+
+Many changes, including bug fixes and documentation improvements, can be
+implemented and reviewed via the normal GitHub pull request workflow.
+
+However, some changes are "substantial", and we ask that these be put
+through a bit of a design process and produce a consensus among the ESLint Technical Steering Committee (TSC).
+
+The "RFC" (request for comments) process is intended to provide a
+consistent and controlled path for new features to enter the project.
+
+## When you need to follow this process
+
+You need to follow this process if you intend to make "substantial"
+changes to any part of the ESLint project or its documentation. What constitutes a
+"substantial" change is evolving based on community norms, but may
+include the following.
+
+    - A new ESLint command line option.
```

Comment:
Ah, I just noticed this looks a little weird with the way GitHub renders markdown. Thoughts on making this a numbered list like the others?

Reply:
I'll check that out.